interface/modules/custom_modules/oe-module-prior-authorizations/public/index.php

```diff
@@ -9,7 +9,6 @@
  */
 
 require_once dirname(__FILE__, 5) . "/globals.php";
-require_once dirname(__FILE__, 2) . '/vendor/autoload.php';
```

Comment:
still will need these to support above proposal

Reply:
OpenEMR build should catch and build autoload I'm pretty certain. May depend on namespace however we should insist use openemr namespace for core modules. Otherwise this is going to get out of hand quickly for admins or really mostly you Brady.
I do believe composer looks at all namespaces and will catch classes in namespace for autoload but I'm going by memory here which isn't so great lately! Help me doctor!:)
Still we don't need zip if in core and don't want a clever user doing a composer install in core tree.